app/src/test/java/com/kickstarter/viewmodels/BackingAddOnViewHolderViewModelTest.kt

```diff
@@ -0,0 +1,76 @@
+package com.kickstarter.viewmodels
+
+import androidx.annotation.NonNull
+import com.kickstarter.KSRobolectricTestCase
+import com.kickstarter.libs.Environment
+import com.kickstarter.mock.factories.ProjectDataFactory
+import com.kickstarter.mock.factories.ProjectFactory
+import com.kickstarter.mock.factories.RewardFactory
+import com.kickstarter.models.Reward
+import com.kickstarter.models.ShippingRule
+import com.kickstarter.ui.data.ProjectData
+import com.kickstarter.viewmodels.BackingAddOnViewHolderViewModel
+import org.junit.Test
+import rx.observers.TestSubscriber
+
+class BackingAddOnViewHolderViewModelTest : KSRobolectricTestCase() {
+    private lateinit var vm: BackingAddOnViewHolderViewModel.ViewModel
+
+    private val backerLimitIsGone = TestSubscriber.create<Boolean>()
+    private val remainingQuantityIsGone = TestSubscriber.create<Boolean>()
+    private val countdownIsGone = TestSubscriber.create<Boolean>()
+    private val shippingAmountIsGone = TestSubscriber.create<Boolean>()
+
+    private fun setupEnvironment(@NonNull environment: Environment ) {
+        this.vm = BackingAddOnViewHolderViewModel.ViewModel(environment)
+        this.vm.outputs.backerLimitPillIsGone().subscribe(this.backerLimitIsGone)
+        this.vm.outputs.remainingQuantityPillIsGone().subscribe(this.remainingQuantityIsGone)
+        this.vm.outputs.deadlineCountdownIsGone().subscribe(this.countdownIsGone)
+        this.vm.outputs.shippingAmountIsGone().subscribe(this.shippingAmountIsGone)
```

Comment:
Same with the items here, a test covering that would be nice :)

Reply:
Done!